scripts/testLeapGrasp.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import JointState
+import numpy as np
+import time
+
+class JointOscillatorNode(Node):
+    def __init__(self):
+        super().__init__('joint_oscillator_node')
+
+        self.publisher = self.create_publisher(JointState, '/dom/command_joint_states', 10)
+
+        # Initialize joint angles
+        self.joint_angles = [0.0] * 16
+
+        # Define angles for grabbing
+        self.grab_angles = np.array([0.0] * 16)
+
+        # Define angles for releasing
+        self.release_angles = np.array([0.0] * 16)
+
+        # # Create a timer to publish joint angles at 60 Hz
+        # self.timer = self.create_timer(1.0 / 60.0, self.publish_joint_angles)
+        
+        # self.grab()  # Call grab to set angles for grabbing
+        # self.release()
+        for _ in range (12):
+            input("Press Enter to Grab")
+            self.grab()  # Call grab to set angles for grabbing
+            # time.sleep(2)  # Simulate grabbing for 5 seconds
+            input("Press Enter to Release")
+            self.release()  # Call release to set angles for releasing
+            # time.sleep(2)  # Simulate releasing for 5 seconds
+            # input("Press Enter to Grab")
+
+
+    def grab(self):
+        # # self.grab_angles[[0, 4, 8]] = 0.0
+        # self.grab_angles[0] = 5.0
+        # self.grab_angles[4] = -3.0
+        # self.grab_angles[8] = -15.0
+        # self.grab_angles[[1, 5, 9]] = 80.0   #Fingers grip
+        # self.grab_angles[[2, 6, 10]] = 30.0
+        # self.grab_angles[[3, 7, 11]] = 15.0     #15
+        # # self.grab_angles[[3, 7, 11]] = 30.0     #15
+
+
+        # self.grab_angles[12] = 100.0
+        # self.grab_angles[13] = 3.0
+        # self.grab_angles[14] = -10.0     #Thumb grip
+        # self.grab_angles[15] = 60.0     #60
+        # # self.grab_angles[15] = 85.0     
+        
+        self.grab_angles[0] = 5.0
```

Comment:
When testing, I dont believe these angles to be correct with the renumbering of the motors

Reply:
fixed